packages/core/tests/controllers/TransactionsController.test.ts

```diff
@@ -0,0 +1,15 @@
+import { describe, expect, it } from 'vitest'
```

Comment:
This test is failing due to a package resolve issue in CI and cannot reproduce it in local. 

Might there be some caching issue since we're using Turbo? I had times that it's causing this when we use remote cache feature

@0xAsimetriq 

- https://github.com/WalletConnect/web3modal/actions/runs/6812592630/job/18525268647?pr=1440
- https://turbo.build/repo/docs/core-concepts/remote-caching

Reply:
Can you try updating `common` package to `3.2.1`?